doc/USERMANAGEMENT.md

```diff
@@ -173,20 +171,181 @@ Now you can go to `import`, paste that template and click `proceed` and the user
 
 #### Add the user manually
 
-You can add a user manually through phpLDAPadmin, after connecting go to `⭐️Create new entry here` :
+You can add a user manually through phpLDAPadmin, after connecting go to `Create new entry here` :
 
 <img src="images/installation/Create_new.gif" alt="Create user"/>
 
 You can for example create a user in the default admin group `cn=admin,dc=ldap,dc=cetic,dc=be`, or create a new group in which you can create new users. 
 
-In this example we are going to create a simple user under the default admin user (which is also a group).
+In this example, we are going to create a simple user under the default admin user (which is also a group).
 
 When you click on  `⭐️Create new entry here`, a new window called `Select a template for the creation process` will show up with all the different entries you can create:
 
 <img src="images/installation/Generic_User_Account.png" alt="Create a new user"/>
 
 Go to `Generic: User Account` and a list of fields will show up. Enter the information about the user you want to create and click `Create Object`.
  
+## Creating groups
+
+The LDAP protocol do not define how programs function either on the server or client, but the messages exchanged between an LDAP server and an LDAP client, to manage your users well you need to know how to create users/groups in the LDAP server and then you need to assign every user/group to the right service or application **through the application's configuration on the `values.yaml` file**.
```

Comment:
do -> does

Split the sentence in 2 after "...and a LDAP client"

Remove "well"